python/mxnet/contrib/serde/_import/__init__.py

```diff
@@ -0,0 +1,26 @@
+# coding: utf-8
+"""import function"""
+import onnx
+from .import_onnx import GraphProto
+
+def import_model(model_file):
```

Comment:
Propose to rename to import_onnx_model

Reply:
Is `import_model` a user level API? If so, why is it registered under a hidden module `_import`?